core/foundation/src/TClassEdit.cxx

```diff
@@ -48,13 +48,20 @@ namespace {
             gInterpreterHelper->ShuttingDownSignal();
       }
    };
-}
+
+   ////////////////////////////////////////////////////////////////////////////////
+   /// Remove the next spaces.
+   void RemoveSpace(std::string_view &s)
+   {
+      while (!s.empty() && s[0] == ' ')
+         s.remove_prefix(1);
+   }
 
 ////////////////////////////////////////////////////////////////////////////////
```

Comment:
If those are brough into the (unnamed) namespace they probably should also follow the indentation of the other members (and/or vice et versa) for consistency.

Reply:
I was thinking about it, but ultimately, it created a lot of noise in the diffs. I wasn't sure if that diff noise wouldn't hide the real changes.

Reply:
Added it now as an [NFC] commit